src/cascadia/TerminalApp/App.cpp

```diff
@@ -445,7 +445,12 @@ namespace winrt::TerminalApp::implementation
         co_await winrt::resume_background();
 
         const auto settingsPath = CascadiaSettings::GetSettingsPath();
-        ShellExecute(nullptr, L"open", settingsPath.c_str(), nullptr, nullptr, SW_SHOW);
+
+        HINSTANCE res = ShellExecute(nullptr, nullptr, settingsPath.c_str(), nullptr, nullptr, SW_SHOW);

```

Comment:
I get the "try something else when the first one fails," but why did you drop the "open" verb?

Also can you leave a comment back to this page https://docs.microsoft.com/en-us/windows/win32/api/shellapi/nf-shellapi-shellexecutea that explains why the result code checking is so horrific for this API?

Reply:
I think dropping the open verb is also documented there:

>**NULL**
> The default verb is used, if available. If not, the "open" verb is used. If neither verb is available, the system uses the first verb listed in the registry.

That seems strictly better. GVim registers the _edit_ verb (which we learned in #1789!)

Reply:
I tried to explain this in the commit message. Gvim only sets up the registry for the "edit" verb so it does not work with the "open" verb. Setting this to NULL solves the issue as well.

> NULL
> The default verb is used, if available. If not, the "open" verb is used. If neither verb is available, > the system uses the first verb listed in the registry.

Reply:
@MiNiksa if you're concerned about the oddities of the ShellExecute return value, it might make sense to use ShellExecuteEx instead because that returns true or false.

As to why it's so odd, [it made more sense in 16 bits.](https://devblogs.microsoft.com/oldnewthing/20060505-08/?p=31293)